scripts/serverless-setup.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Create and commit .sam-params-<env> file as SAM deploy parameters
+# The cert ARN and hostedZoneID are not sensitive but requires list all permissions in IAM
+# therefore are created here instead of granting the CI user those permission to fetch on the fly
+# and they should not change often
+
+if [[ "$ENVIRONMENT" == "stage" ]]; then
+  BACKEND_DOMAIN_PREFIX=$stagingBackendSubdomain
+  DOMAIN=$stagingHostRoot
+  FILE_NAME=".sam-params-stage"
+elif [[ "$ENVIRONMENT" == "prod" ]]; then
+  BACKEND_DOMAIN_PREFIX=$productionBackendSubdomain
+  DOMAIN=$productionHostRoot
+  FILE_NAME=".sam-params-prod"
+fi
+
+HOSTED_ZONE_ID=$(aws route53 list-hosted-zones  --query "HostedZones[?Name == '${DOMAIN}.'].Id"  | jq -r '.[0]' | sed 's/\/hostedzone\///')
```

Comment:
Similar to below you can get rid of the `jq` call using ` | [0]`   and `--output text` though you'd still need to split that string though. (So weird that they return it in that format in the first place...)